### ISSUE TITLE
A11y_AzureDataStudioAzureArcExtension_Sql Instance 1_Connection Strings_DeveloperTool: No name property is provided for the "ADO.NET Copy" button

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15756]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AzureArcExtension_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#DevTools;#WCAG4.1.2;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio -Azure Arc Extension
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "Sqlinstance1" and open manage and tab to "Connection Strings" page
5. Tab to "ADO.NET" respective "Copy" button hit "ctrl + shift + i" and check if it has name provided.

### Actual:
No name property is provided for the "ADO.NET Copy" button.
similar issue is observed in the scenario given below:

1. PG1_Properties
2. PG1_Connection Strings
3. Sql Instance 1_Compute + Storage - Name of the spin buttons not provided
4. PG1_Compute + Storage - Name of the spin buttons not provided

### Expected:
Name property for the "ADO.NET Copy" button should be assigned according to it's label which is "ADO.NET Copy"

### User Impact:
Screen Reader users will not get the proper information about the button

**Comment by krrish2021** [comment]
Closing as not repro
![Screenshot (180)](https://user-images.githubusercontent.com/85863543/150957934-15dd6012-9cce-4993-a375-e337cf8c95b2.png)

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
